src/app/layout.tsx

```diff
@@ -8,6 +8,7 @@ import Footer from "@/app/_components/footer";
 import type { Metadata } from "next";
 import Container from "@/app/_components/container";
 import { Navbar } from "@/app/_components/navbar";
+import { Banner } from "./_components/banner";
```

Comment:
Use absolute paths with @/ when possible. It lets us more easily move files around in the future if we need to

Reply:
Got it. I was just letting vscode write the imports for me.

Reply:
fixed

Reply:
Normally vscode does it but it's not doing it here, wonder if I there's some config we need to set

Reply:
Looks like the setting is `"typescript.preferences.importModuleSpecifier": "non-relative"`, we can put that in a `.vscode/settings.json`